### ISSUE TITLE
Update Monolog to 3.x

### ISSUE THREAD

**Issue by devmount** [Geotab/mygeotab-php#7]
As already done for #4, my application (Laravel 10, PHP 8.1) needs an upgrade to `monolog/monolog ^3.0`. Otherwise there are dependency issues:

```
geotab/mygeotab-php 1.2.0 requires monolog/monolog ^2.0 -> satisfiable by monolog/monolog[2.0.0-beta1, ..., 2.x-dev]
```

Any plans to update this? Do you need help with a PR?

**Comment by colonelchlorine** [comment]
Fixed in [5c248b9](https://github.com/Geotab/mygeotab-php/commit/5c248b9c40bc30e841c2013eed8afd91f146e204). Thanks for issue creation.

**Comment by devmount** [comment]
Wow, thank you for the quick fix 🙏🏻
